### PR TITLE
Update Travis config to use industrial_ci. Add .rosintall

### PR DESCRIPTION
**Pull request by 130s**
- For Travis for now this tests `kinetic` only (given that we'll use this setting from kinetic onward)
- Adding `.rosinstall` for downstream packages (that depend on packages in `moveit` repo). This is not like I want to use from CI (at least for now), but it'll be useful for the end users who want to build from source. And since `moveit` repo will become a "core" repo for MoveIt!, I think it makes sense to store a file like this here.

### CODE REVIEW

**Comment by 130s** [comment]
@davetcoleman Would you enable Travis at https://travis-ci.org/profile/davetcoleman for this repo?

File: .rosinstall.downstream [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- git: { local-name: ros-planning/moveit_commander, uri: https://github.com/ros-planning/moveit_commander.git, version: indigo-devel }
+- git: { local-name: ros-planning/moveit_docs, uri: https://github.com/ros-planning/moveit_docs.git }
+- git: { local-name: ros-planning/moveit_msgs, uri: https://github.com/ros-planning/moveit_msgs.git }
+- git: { local-name: ros-planning/moveit_planners, uri: https://github.com/ros-planning/moveit_planners.git }
+- git: { local-name: ros-planning/moveit_plugins, uri: https://github.com/ros-planning/moveit_plugins.git }
+- git: { local-name: ros-planning/moveit_robots, uri: https://github.com/ros-planning/moveit_robots.git }
+- git: { local-name: ros-planning/moveit_setup_assistant, uri: https://github.com/ros-planning/moveit_setup_assistant.git }
```
Comment (davetcoleman) [inline .rosinstall.downstream]: commander, planners, plugins, and setup_assistant are already in this repo - why are you listing them here also? these repos will be deprecated once we switch to single ``moveit``

**Comment by davetcoleman** [comment]
- If we switch this to kinetic now, will it still build? I was assuming we are missing kinetic dependencies but I haven't checked in a while
- Travis has been enabled for the past week - why do you think it is not?
- I think we need to remove some of those repos from the .rosinstall file before I can merge this

**Comment by davetcoleman** [comment]
I'll go ahead and merge this since I want to push a new version of this repo out today

**Comment by davetcoleman** [comment]
@130s I renamed the .rosinstall so that Travis will pull the other required moveit repos when using Industrial CI - does this look right?
https://github.com/davetcoleman/moveit/commit/593735150ae9c8d9b724d572dadf940e56ab8006

File: .rosinstall.downstream (hunk 1 above)
Comment (130s) [inline .rosinstall.downstream]: My bad, I'll open a new PR to fix this (I copied this .rosinstall file from somewhere and admittedly wasn't careful enough to review the content).

**Comment by 130s** [comment]
>    If we switch this to kinetic now, will it still build? I was assuming we are missing kinetic dependencies but I haven't checked in a while

Turned out it doesn't build with the .travis.yml I sent a PR, because some DEB of the dependencies, notable [moveit_msgs](https://travis-ci.org/davetcoleman/moveit/jobs/139257926#L5884), are not yet available. I'll open a PR to add source build option.

>   Travis has been enabled for the past week - why do you think it is not?

Oh, I see. 
(It didn't start working for me. I don't know, but there are times where github doesn't hook Travis, which seems to users that Travis wasn't activated for the repo.)

**Comment by davetcoleman** [comment]
@130s I made a bunch of changes and added both a moveit.rosinstall file (for users) and .travis.rosinstall file (for pulling down source dependencies like moveit_msgs). It seems like its working now but I know of at least one test error in moveit_planning/ompl where this combined repo suddenly is running all the packages tests - even some that have not worked in a long time

**Pull request merged**